### ISSUE TITLE
Add an option to ignore the new "application" keyword

### ISSUE THREAD

**Issue by jdubois** [jhipster/jhipster-core#204]
The new application generation from the JDL is totally awesome, but it causes me 2 issues with https://start.jhipster.tech

- The current way it works is that people generate the application with an Angular form, and then add the entities: so that won't work well with the new "application" keyword.
- Most importantly: with the "path" keyword, they can generate applications anywhere on the filesystem, and we can't have this on https://start.jhipster.tech - they could overwrite some files, for example

So for the moment I would just like an option flag, like `--ignore-application`, to remain with the old behavior. Otherwise I won't be able to have this on https://start.jhipster.tech quickly.

**Comment by deepu105** [comment]
@MathieuAA I was gonna ask you about the Path as well, do we really need it as it could lead to many edge cases and bug reports, cant we follow what we do now, generate in the folder you execute the command from. For microservice when all apps are in single file we can assume its executed from a parent folder and create folders within that and apps in those folders. WDYT?

@jdubois yes lets add that flag to import-jdl for now

**Comment by deepu105** [comment]
@jdubois its easier to do `--ignore-application` on the sub generator than in JH core

**Comment by jdubois** [comment]
Oh yes you are correct @deepu105 -> the flag should be in the sub-generator, but then JH Core must also understand this

**Comment by deepu105** [comment]
Not necessarily, since the app generation and writing of json files happens in the sub gen and you just need to skip those parts so that regardless of what JHcore returns you only generate entities

**Comment by deepu105** [comment]
To be exact the `shouldGenerateApplications` method in the sub Gen needs to return false when we specify the flag

**Comment by MathieuAA** [comment]
I agree with Deepu. This solution is good enough.
